config/config.go

```diff
@@ -131,3 +142,67 @@ func InitChains(ctx *Context, homePath string, debug bool) error {
 
 	return nil
 }
+
+func (c *Config) InitConfig(ctx *Context, homePath, configPath string, debug bool) error {
```

Comment:
It is better to use `func (c *Config) InitConfig(ctx *Context, configPath string, debug bool) error`.
`configPath` should be determined by a caller of this function.

Reply:
https://github.com/dongrie/yui-relayer/blob/path-config-update/config/config.go#L159-L161
https://github.com/dongrie/yui-relayer/blob/path-config-update/config/config.go#L138-L140
Because it uses homePath internally like this, the parameter uses homePath